source/identical.js

```diff
@@ -27,5 +27,23 @@ import _curry2 from './internal/_curry2.js';
  *      R.identical(0, -0); //=> false
  *      R.identical(NaN, NaN); //=> true
  */
-var identical = _curry2(_objectIs);
```

Comment:
IIRC `_curry2` was the only problem as it was trying to work out if a placeholder was given in place of an argument? I didn't check `_objectIs`; have you checked that it won't try to access things that cross-origin windows would forbid? I think leaving a comment would be helpful as I can see how tempting it would be for someone in the arbitrary future to rollback that change thinking **Why didn't they use `_curry2`?!**

Reply:
Added a comment why `_curry2` cannot be used in the source file.